src/main/java/org/mskcc/cmo/messaging/impl/JSGatewayImpl.java

```diff
@@ -297,4 +296,43 @@ public PublishingQueueTask(String msgId, String subject, Object payload) {
             this.payload = payload;
         }
     }
+
+    @Override
+    public String request(String subject, Object message) {
```

Comment:
Should i return a string message here or maybe a messageConsumer?

Reply:
MessageConsumer prob makes sense? or maybe the NATS `Message` java object that we are setting the `reply` to